evap/evaluation/management/commands/ts.py

```diff
@@ -54,8 +54,8 @@ def handle(self, *args, **options):
     def run_command(self, command):
         try:
             subprocess.run(command, check=True)  # nosec
-        except FileNotFoundError:
-            print(f"Could not find {command[0]} command", file=self.stderr)
```

Comment:
As it turns out, the issue was these lines in `ts.py` and `scss.py`, as they merely printed the error without changing the return code

Reply:
I raise a CommandError instead and adopted the tests accordingly